inference-engine/samples/common/utils/include/samples/csv_dumper.hpp

```diff
@@ -18,7 +18,7 @@
  */
 class CsvDumper {
     std::ofstream file;
-    std::string filename;
+    std::string filename = "";
```

Comment:
string is initialized by a default ctor. Why do we need to do it explicitly?

Reply:
to avoid the Klocwork issue or I can ignore it.

Reply:
we need to mark that issue as false alarm

Reply:
ok, I closed PR